Dockerfile

```diff
@@ -66,9 +66,18 @@ VOLUME /home/jekyll
 WORKDIR /home/jekyll
 
 # kabanero.io gem dependencies
-COPY ./scripts /home/jekyll/scripts
-COPY Gemfile /home/jekyll
+
+
+# kabanero.io custom gems
 COPY ./gems /home/jekyll/gems
+RUN pushd gems/ol-asciidoc \
+    && gem build ol-asciidoc.gemspec \
+    && gem install ol-asciidoc-0.0.1.gem \
+    && popd
```

Comment:
don't think you need this RUN command the Gemfile should include this gem and install it via the build gem dependencies script

Reply:
gotcha I just put ill put it back